sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UninvokedEventDeclaration.cs

```diff
@@ -93,6 +93,25 @@ public void M()
         }
     }
 
+    // See https://github.com/SonarSource/sonar-csharp/issues/1219
+    public class EventAccessor
+    {
+        private event EventHandler privateEvent;
+
+        public event EventHandler PublicEvent
+        {
+            add { privateEvent += value; }
+            remove { privateEvent -= value; }
+        }
+
+        protected void OnEvent1()
+        {
+            PublicEvent += (s, o) => { }; // Need to use the event to be able to trigger issues
```

Comment:
IF the event declaration is not used we don't report, hence we need this line.

Reply:
If that's not used do you know if we have another rule triggering (like unused member or something like that)? I actually think we should always report because this rule is about dead code so not subscribing + not raising = issue.

WDYT?